### PR TITLE
BP-2553,BP-2554

### PR DESCRIPTION
**Pull request by Rinor12010**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@buckaroo/buckaroo_sdk",
-    "version": "1.0.0",
+    "version": "0.9.0",
```
Comment (harli91) [inline package.json:3]: why did you change the version?

File: tests/BuckarooClient.test.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export default initializeBuckarooClient(
         websiteKey: process.env.BPE_WEBSITE_KEY || ''
     },
     {
-        mode: process.env.BPE_MODE === 'live' ? 'live' : 'test',
+        mode: 'test',
```
[the remark this answers is not in the document]
Comment (Rinor12010) [reply tests/BuckarooClient.test.ts]: fixed it dont remeber why I changed it just forgot to change it back

File: package.json (hunk 1 above)
Comment (Rinor12010) [reply package.json:3]: The current release version is 0.9-Beta

**Review by harli91 — APPROVED** [review]
